src/surrogates/src/InterpolationSurrogateIOASCII.C

```diff
@@ -59,7 +59,7 @@ namespace QUESO
 
     // Only processor 0 does the reading.
     // We'll broadcast the data as needed
-    if( env.fullRank() == root )
+    if( env.fullRank() == (int)root )
```

Comment:
In libMesh we have a proc_id_type, but since we're only supporting MPI for distributed parallelism everything ends up cast to int in the end; we just let users save a few megabytes if they want to limit themselves to 65K cores.